openedx/core/djangoapps/courseware_api/tests/test_notes_visibility.py

```diff
@@ -0,0 +1,36 @@
+from unittest.mock import Mock  # lint-amnesty, pylint: disable=wrong-import-order
+from lms.djangoapps.courseware.tests.factories import StudentInfoFactory
+from lms.djangoapps.courseware.tests.helpers import LoginEnrollmentTestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase  # lint-amnesty, pylint: disable=wrong-import-order
+from xmodule.modulestore.tests.factories import CourseFactory  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
@VanDavv Why are the `# lint-amnesty, pylint: disable=wrong-import-order` necessary here?